src/mailing/sender.js

```diff
@@ -43,5 +45,34 @@ async function sendTestEmail(to, username) {
   });
 }
 
-// Export the transport and the sendTestEmail function
-module.exports = { sendTestEmail };
+async function sendForgotPasswordEmail(toEmail, resetToken) {
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
```

Comment:
Unless you're explicitly trying to test by sending from a gmail account, what we currently have in other branches is using `process.env.EMAIL_SERVICE` which is `'Outlook'` to match the email we're sending from (`codethedream.practicum.team4@outlook.com`). Are you testing from a different address? If so, that's totally fine but let's make sure to commit the version that everyone can use for testing.